server/backend/db_backend.go

```diff
@@ -465,7 +474,10 @@ func (self *MongoBackend) isContract(address string) bool {
 	var c models.Address
 	err := self.mongo.C("Addresses").Find(bson.M{"address": address}).Select(bson.M{"contract": 1}).One(&c)
 	if err != nil {
-		self.Lgr.Debug("isContract", zap.String("Address", address), zap.Error(err))
+		if err == mgo.ErrNotFound {
+			return false
+		}
+		self.Lgr.Error("Failed to check if contract", zap.String("address", address), zap.Error(err))
```

Comment:
Many of these debug calls seem worth always logging, as long as they are not just 'not found' errors.